sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -218,5 +218,5 @@ void saveBulkEditWithFacilities(
 
 	void updateExternalData(List<ExternalDataDto> externalData) throws ExternalDataUpdateException;
 
-	void updateCompletenessTask();
+	int updateCompletenessTask();
```

Comment:
There is still the superfluous "Task" suffix

Reply:
removed